lib/SIL/SILConstants.cpp

```diff
@@ -485,6 +484,61 @@ ArrayRef<SymbolicValue> SymbolicValue::getAggregateValue() const {
   return ArrayRef<SymbolicValue>(value.aggregate, aux.aggregate_numElements);
 }
 
+//===----------------------------------------------------------------------===//
+// Unknown
+//===----------------------------------------------------------------------===//
+
+namespace swift {
+namespace internal {
+/// When the value is Unknown, this contains information about the unfoldable
+/// part of the computation.
+struct UnknownInfo {
```

Comment:
It is admittedly a trivial micro optimization, but using a trailing array for this would save some space and be more uniform with the other representation types.  Not required of course, there should only be a few unknown types in the allocator.   If you choose not to do this, please add a comment explaining why so it is known to be an explicit decision, thanks!

Reply:
Done. I admit I haven't used llvm::TrailingObjects lots so I wasn't used to the idiom. I think I get it now.